ivanyu199012/6.-Package_Electron_n_django_app
# Overall Code Adjustments

@ivanyu199012 I made the following adjustments and have ensured the package still behaves correctly in building and running the code. Please take a look.

package.json
- Bump from Node 14 to Node 16
- Add engineStrict and engine parameters to package.json
- Add maker for dmg archives (for macOS specifically. comment out if unnecessary)
- Bump all packages to latest version
- Update Forge syntax for plugins to latest .naverc
- Add naverc file to allow the use of nave for setting the node version .eslintrc.json
- Change styling to use typescript standards popularized by airbnb. This is an industry standard .editorconfig
- Add an editorconfig file to set the standard for line ending, tab size for for all files Typescript files
- Fix to eslint standards
- Reorder code to ensure it follows Javascript call order index.html
- Remove superfluous body tag
- Add required lang attribute to HTML tag Python
- Fix to PEP-8 formatting standards
- Add Prettier (via eslint)
